Unify the log level for wpt server

qiuzhong opened the pull request:
When `--log-*-level` option is used in `./wpt run` command line to set
the log level, it should be applied to the wpt server as well.
Fixed: #10512

qiuzhong committed:
Unify the log level for wpt server

When `--log-*-level` option is used in `./wpt run` command line to set
the log level, it should be applied to the wpt server as well.
Fixed: #10512

qiuzhong committed:
Remove unused imports

jgraham reviewed (changes requested):
That code can't just be deleted. Not treating server errors as a level higher than warning is important for gecko infra because we occasionally get tests that create errors in the handler script which would, if allowed to log at error level, cause jobs to fail.

qiuzhong committed:
Set the same log level for wpt server as the wpt runner.

qiuzhong committed:
Add the verify option when setup log for test server

Skip the log level filter setup for test server when the `verify`
option is added.

qiuzhong committed:
Fix test cases error for setting up the log level filter

foolip commented:
@qiuzhong Travis CI is green now, looks you rebased to trigger it again. Is this now waiting for re-review by @jgraham or what remains to be done?

qiuzhong commented:
@foolip , I fixed some the CI errors. Yes, it is now waiting for re-reviewing by the reviewers.

qiuzhong commented:
@jgraham, @gsnedders , @jugglinmike , would you please take a look at this?
